core/impl-base/src/main/java/org/jboss/arquillian/core/impl/ManagerImpl.java

```diff
@@ -640,7 +663,9 @@ private String getEventName(Object object)
    
    private String calcDebugPrefix()
    {
-      int size = eventStack.get().size();
+
+      final Stack<Object> objects = eventStack.get();
+      int size = objects.size();
```

Comment:
What does it change? It's still not null-safe isn't it? (even though for now on you only call it from places where you initialize the stack)